src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -351,6 +351,20 @@ await AssertIdTokenValidIfExisting(response.IdToken, request.ClientId, request.S
             return response;
         }
 
+        /// <inheritdoc/>
+
+        public Task<string> RevokeRefreshTokenAsync(RevokeRefreshTokenRequest request, CancellationToken cancellationToken = default)
+        {
+            if (request == null)
+                throw new ArgumentNullException(nameof(request));
+
+            return connection.SendAsync<string>(
```

Comment:
We need to consider the Client Assertion, like is done here: https://github.com/auth0/auth0.net/blob/7a323aee7a15eb9007ebb6db21e43bea013c3e85/src/Auth0.AuthenticationApi/AuthenticationApiClient.cs#L140